cmake/modules/OmrDDRSupport.cmake

```diff
@@ -31,6 +31,22 @@ include(ExternalProject)
 # Make sure we have performed platform detection
 include(OmrPlatform)
 
+# DDR is not supported when cross-compiling. Here we automagically
+# enable DDR for native builds and disable for cross-compilation.
+#
+# If OMR_DDR is explicitly set, respect that request (if possible).
+if(NOT DEFINED OMR_DDR)
+	if(CMAKE_CROSSCOMPILING)
+		set(OMR_DDR OFF)
+	else()
+		set(OMR_DDR ON)
+	endif()
+endif()
+
+if(OMR_DDR AND CMAKE_CROSSCOMPILING)
+	message(FATAL_ERROR "DDR is not supported when cross-compiling.")
+endif()
+
```

Comment:
@keithc-ca @dnakamura tagging for a review here. Is this implementation ok with you?